spec/Section 6 -- Execution.md

```diff
@@ -357,19 +346,151 @@ Unsubscribe(responseStream):
 
 - Cancel {responseStream}.
 
-## Executing Selection Sets
+## Executing the Root Selection Set
```

Comment:
this new header name seems like it should not be at `##` level anymore, but the sub-algos still matter. 

Perhaps this top level section should remain and explain the high level idea, and then have these be sub-headers